pkg/apis/serving/v1alpha1/kfservice_defaults_test.go

```diff
@@ -27,7 +27,7 @@ import (
 func TestTensorflowDefaults(t *testing.T) {
 	g := gomega.NewGomegaWithT(t)
 	kfsvc := TFExampleKFService.DeepCopy()
-	kfsvc.Spec.Canary = &CanarySpec{ModelSpec: *kfsvc.Spec.Default.DeepCopy()}
+	kfsvc.Spec.Canary = kfsvc.Spec.Default.DeepCopy()
 	kfsvc.Spec.Canary.Tensorflow.RuntimeVersion = "1.11"
```

Comment:
Do we need to any additional scritps in kfserving/hack folder to regen deepcopy files?

Reply:
deepcopy files should be covered already since it is under pkg.